app/services/create_proposed_organisation_edit.rb

```diff
@@ -36,6 +36,7 @@ def merge_in_non_published_fields
 
   def send_email_to_superadmin_about_org_edit
     superadmin_emails = user_klass.superadmins.pluck(:email)
```

Comment:
@tansaku, so we are using "extract method" here. grabbing a chunk of code and making it its own method. Now, the new method after your suggestion will look like:

`def self.superadmin_emails`
  `superadmin_emails = user_klass.superadmins`
`end`

this is my question(s):
1. how do we call the method we created above from the source method:

`def send_email_to_superadmin_about_org_edit`
    `// how to call the method on user.rb??`
    `mailing = mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails)`
    `mailing.deliver_now`
  `end`

Finally,
how do we deal with the *existing* cukes when we add a new method? do we need to write new tests to take into account the new method. I believe the cukes will still pass but we are re-arranging the functionality of code. 

also, what tools do we use to analyze code? Reek, etc and what effective these tools are in dealing with refactoring.